src/main/java/cam72cam/immersiverailroading/items/ItemPaintBrush.java

```diff
@@ -27,4 +41,81 @@ public List<CreativeTab> getCreativeTabs() {
 		return Collections.singletonList(ItemTabs.MAIN_TAB);
 	}
 
+	@Override
+	public List<String> getTooltip(ItemStack stack) {
+		PaintBrushMode pbm = new Data(stack).mode;
+		if (pbm == null) {
```

Comment:
AFAICT this is an impossible condition

Reply:
I had copiedthat from I think ItemManual. will remove.